### PR TITLE
feat: cc-0075 add border component to courts

### PR DESCRIPTION
**Pull request by EmilyGong111**
Add border element to courts, when borderLength changes, the border gonna show corresponding width. 
Currently, proFullCourt default borderLength is 1000, others are 0.

Default is 0
Screen Shot 2022-06-30 at 12 56 25 pm

When the border is 300
Screen Shot 2022-06-30 at 12 57 29 pm

### CODE REVIEW

File: src/components/BasketballCourt/Border.tsx [hunk 1]
```diff
@@ -6,11 +6,19 @@ import { changeTileColor } from "@/store/reducer/tileSlice";
...
-  );
+const Border: React.FC<BorderProps> = ({
+  startPoint,
+  borderLength,
+  courtAreaXLength,
+  courtAreaYLength,
+}) => {
+  // const { courtAreaXLength, courtAreaYLength} = useStoreSelector(
+  //   (state) => state.courtSize
+  // );
```
Comment (tatoMa) [inline src/components/BasketballCourt/Border.tsx:21]: If these are needed please add them to TODO.

**Review by tatoMa — APPROVED** [review]
LGTM

**Review by stkwn — APPROVED** [review]
Tested and working well.

File: src/components/BasketballCourt/Border.tsx (hunk 1 above)
Comment (EmilyGong111) [reply src/components/BasketballCourt/Border.tsx:21]: Get it！thanks~

**Pull request merged**